src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { Todo } from '../../types/Todo';
+import { TodoItem } from '../TodoItem/TodoItem';
+
+type Props = {
+  todos: Todo[];
+  tempTodo: Todo | null;
+  isIncludeId: number[];
+  onDeleteItem: (id: number) => void;
+};
+
+export const TodoList: React.FC<Props> = ({
+  todos,
+  tempTodo,
+  isIncludeId,
+  onDeleteItem,
+}) => {
+  return (
```

Comment:
```suggestion
}) => (
```

try to check all your components, and remove `return` where necessary